build.sbt

```diff
@@ -10,32 +10,29 @@ scalacOptions := Seq(
   "-Xfatal-warnings"
 )
 
-
-// We must include both AWS SDK V1 and V2 to enable the use of latest
-// Scanamo whilst avoiding overhauling the whole app to V2.
-val awsSdkVersion = "1.12.130"
-val awsSdkVersionV2 = "2.31.19"
+val awsSdkVersionV2 = "2.34.6"
 val jacksonVersion = "2.19.2"
+val playSecretRotationVersion = "15.1.0"
 
 libraryDependencies ++= Seq(
   jdbc,
   ws,
-  "com.gu" %% "pan-domain-auth-play_3-0" % "7.0.0",
-  "com.gu.play-secret-rotation" %% "aws-parameterstore-sdk-v1" % "7.1.1",
-  "com.gu.play-secret-rotation" %% "play-v30" % "7.1.1",
+  "com.gu" %% "pan-domain-auth-play_3-0" % "11.0.0",
```

Comment:
`12.0.0` is now available, which bumps a few dependencies: https://github.com/guardian/pan-domain-authentication/pull/200